rust/tvm/src/ir/expr.rs

```diff
@@ -32,12 +32,14 @@ use super::span::Span;
 #[type_key = "Expr"]
```

Comment:
would it make sense to make a macro for the boilerplate?

Reply:
This is a macro 🤯

Reply:
I see. I just see the span boilerplate and thought you can refactor them. cool in this case then.